app/ui/src/app/integration/edit-page/flow-view/flow-view-step.component.html

```diff
@@ -19,7 +19,7 @@
 
   <!-- Icon Progress Line -->
   <div class="col-md-3 progress-line">
-    <p [class]="'icon ' + getParentActiveClass() + ' ' + getMenuCompleteClass() + ' ' + getConnectionClass()"
+    <div [class]="'icon ' + getParentActiveClass() + ' ' + getMenuCompleteClass() + ' ' + getConnectionClass()"
```

Comment:
Let's try to stick to the Angular spec and use `[ngClass]` instead. Relying on non-supported syntaxes just because it works might lead to unexpected behaviors in the future (in this case, `[class]` just works because it is a combination of Angular's current template engine, which will be replaced by IvyTemplate in Angular 6, and the default W3C spec).

Reply:
@deeleman done!

Would it make sense to create an issue to address all of the cases where `[class]` is currently used? Looks like it only occurs in these 10 files

    cmichael:src cmichael$ grep -ri '\[class\]' *
    app/app.component.html:               [class]="productBuild ? 'navbar-brand-icon product' : 'navbar-brand-icon'"
    app/app.component.html:               [class]="productBuild ? 'navbar-brand-name product' : 'navbar-brand-name'"
    app/customizations/tech-extensions/import/tech-extension-import.component.html:          <div [class]="error.level" [innerHtml]="error.message">
    app/integration/integration_detail/integration-description.component.html:              <div [class]="'step-line ' + getStepLineClass(index)"></div>
    app/integration/edit-page/edit-page.component.html:    <div [class]="getPageRow()">
    app/integration/edit-page/edit-page.component.html:      <div [class]="getSidebarClass()">
    app/integration/edit-page/flow-view/flow-view.component.html:    <div [class]="containerClass"
    app/integration/edit-page/step-configure/step-configure.component.html:    <div [class]="getToolbarClass()">
    app/integration/edit-page/step-configure/step-configure.component.html:          <div [class]="error.class"
    app/dashboard/dashboard_integrations/dashboard-integrations.component.html:                    <span [class]="'label ' + getLabelClass(integration)">